checkstyle/import-control.xml

```diff
@@ -320,6 +320,7 @@
     <allow pkg="javax.rmi.ssl"/>
     <allow pkg="kafka.utils" />
     <allow pkg="scala.collection" />
+    <allow pkg="org.apache.kafka"/>
```

Comment:
It said `Disallowed import `

Reply:
Not very helpful, it must have said that following class is not allowed to import. So we can see what package that class is in and just whitelist that specifically instead of complete `kafka` package. You can see this example on lines above.